docs/pages/setup/admin/adding-nodes.mdx

```diff
@@ -74,36 +112,53 @@ auth_service:
 ```
 </Details>
 
-### Using Node invitation tokens
+### Start your Node with the invite token and CA pin
+
+A CA pin needs to be passed to new Nodes when they're starting for the first
```

Comment:
This is a little confusing, because we say it "needs to be passed to new nodes" and then immediately say it doesn't need to be provided to join a cluster.